crates/astria-core/Cargo.toml

```diff
@@ -37,6 +37,7 @@ penumbra-proto = { workspace = true }
 prost = { workspace = true }
 rand = { workspace = true }
 serde = { workspace = true, features = ["derive"], optional = true }
+serde_json = { workspace = true }
```

Comment:
Will this change force us to always activate serde for all of `astria-core`? After all, the memo serde impls are feature gated I think.

Reply:
I tried to address this in 84b00ef. Let me know if this is what you had in mind!